### ISSUE TITLE
Install IsardVDI with no dockers

### ISSUE THREAD

**Issue by samuak8** [isard-vdi/isard#152]
Hi, we a are a couple of students at the High School Puig Castellar, and we want to try to install IsardVDI without any docker, could you kindly provide the list of packets that we should install.

Thanks in advance!

**Comment by NefixEstrada** [comment]
Hello!

First of all thanks for your interest of the project! It really means a lot to see people getting interested in the project. Why would want to install IsardVDI without Docker? Do you have any specific reason or it's just because you want to try it?

Isard is made of lots of moving parts and services, which are difficult to configure. Because of that, we decided to go full Docker to simplify the process. That being said, if you **really** want to try running Isard without Dockers (which we don't recommend), you can see all the containers that need to be started in the [docker-compose.yml](https://github.com/isard-vdi/isard/blob/master/docker-compose.yml) and see how is built every image inside the [dockers folder](https://github.com/isard-vdi/isard/tree/master/dockers).

If you still have questions, feel free to ask them below!

**TL;DR**: Don't run IsardVDI without Dockers, check the dependecies on the Dockerfiles inside the [dockers folder](https://github.com/isard-vdi/isard/tree/master/dockers)

**Comment by samuak8** [comment]
Hello Nefix! It's our final grade work, and our teacher told us about to perform it without the use of any docker, just using an Ubuntu Server, and finally we want to make a unique script to install Isard and every part and service in one step. We are conscious that is a crazy task but we want to try it.

Thanks for your atenttion!

**Comment by NefixEstrada** [comment]
I see! That'd be nice to have! I'd start with identifying all the different parts of Isard and understanding what each one does. Once it's clear, you'd have to install all the dependencies for each one and write the SystemD configuration files for each service. In case you run into some issue or don't know how to do something don't hesitate and ask!

Good luck with your work!

**Comment by samuak8** [comment]
I don't know to start, but we'll see.

Thanks!

**Comment by samuak8** [comment]
Hello Néfix,

I have a question, Using the docker, what should I do for connect remotely
to IsardVDI.
I mean to accept connections of machines in the same network.

Thanks a lot!

**Comment by NefixEstrada** [comment]
Hello!

Once you have all the dockers running (with `docker-compose up -d`), you should be able to connect to IsardVDI just by entering the IP address of the machine where the dockers are running.

```
+-----------------------+                 +--------------------+
|                       |                 |                    |
|       Server          |                 |       Client       |
|                       |                 |                    |
|   (IsardVDI dockers   |                 |  (Open the browser |
|   running here)       +<----------------+  and access to the |
|                       |                 |  server IP)        |
|   192.168.1.23/24     |                 |                    |
|                       |                 |  192.168.1.54/24   |
|                       |                 |                    |
+-----------------------+                 +--------------------+
```

**Comment by samuak8** [comment]
Hi Néfix,

All dockers are running, but we cannot connect to the IsardVDI server
through the network



+-----------------------+                 +--------------------+
|                       |                 |                    |
|       Server          |                 |       Client       |
|                       |                 |                    |
|   (IsardVDI dockers   |                 |  (Open the browser |
|   running here)       +< NOT AVAILABLE  +  and access to the |
|                       |                 |  server IP)        |
|   192.168.15.180/24 <http://192.168.1.23/24>   |                 |
                 |
|                       |                 |  192.168.15.143/24
<http://192.168.1.54/24> |
|                       |                 |                    |
+-----------------------+                 +--------------------+

**Comment by NefixEstrada** [comment]
Hello!

Sorry for the late response. Are you still experiencing this issue?

**Comment by samuak8** [comment]
Yes i'm still having this issue.

**Comment by jvinolas** [comment]
Hi,
The client will use the client viewer address set up in isard-hypervisor. Please check that there you have the correct IP address where isard-hypervisor is running: https://isardvdi.readthedocs.io/en/latest/admin/faq/#viewers

**Comment by samuak8** [comment]
We put the correct IP, but anyway we can not access.

[image: image.png]

[image: image.png]

**Comment by NefixEstrada** [comment]
This seems an issue related with the firewall on the server. Could you please check that the firewall is disabled or that the traffic at the ports `80` & `443` gets through the firewall

**Comment by samuak8** [comment]
Was that the issue, now works. Thanks for your support!
